tests/utils.js

```diff
@@ -21,7 +21,14 @@ export function removeWhitespace(str) {
 }
 
 export function getRules(style) {
-  return [...style.sheet.cssRules]
+  const rulesArr = []
+  const {cssRules} = style.sheet
+  for (const key in cssRules) {
```

Comment:
Maybe better `for(;;)`?

Reply:
hah, for in failed in safari ... good catch